ngraph/frontend/onnx/frontend/src/op/qlinear_matmul.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Disabled in CMakeList
+// Update to higher opset required
+
+#include "op/qlinear_matmul.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "dequantize_linear.hpp"
+#include "exceptions.hpp"
+#include "matmul.hpp"
+#include "ngraph/opsets/opset6.hpp"
+#include "onnx_import/core/null_node.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector qlinear_matmul(const Node& node) {
+    const OutputVector& inputs = node.get_ng_inputs();
+
+    auto a = inputs.at(0);
+    auto a_scale = inputs.at(1);
+    auto a_zero_point = inputs.at(2);
+    auto b = inputs.at(3);
+    auto b_scale = inputs.at(4);
+    auto b_zero_point = inputs.at(5);
+    auto y_scale = inputs.at(6);
+    auto y_zero_point = inputs.at(7);
```

Comment:
There is no need to make local copies of these inputs. I'd rather see `const auto &` here.

Reply:
Done